### PR TITLE
yum-config-manager do not enable subscription-manager repos

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

**Review by dLobatog — COMMENTED** [review]
@jhutar What is the difference?

**Comment by jhutar** [comment]
Hello @dLobatog . `yum-config-manager` do not work on subscription manager repos:

```
[root@sat630 ~]# yum repolist
Loaded plugins: product-id, search-disabled-repos, subscription-manager
repo id                                                                                                       repo name                                                                                                                 status
rhel-7-server-rpms/7Server/x86_64                                                                             Red Hat Enterprise Linux 7 Server (RPMs)                                                                                  18,263
rhel-server-rhscl-7-rpms/7Server/x86_64                                                                       Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server                                                    9,429
repolist: 28,003
[root@sat630 ~]# yum-config-manager --disable rhel-7-server-rpms
[...]
[root@sat630 ~]# echo $?
0
[root@sat630 ~]# yum repolist
Loaded plugins: product-id, search-disabled-repos, subscription-manager
repo id                                                                                                       repo name                                                                                                                 status
rhel-7-server-rpms/7Server/x86_64                                                                             Red Hat Enterprise Linux 7 Server (RPMs)                                                                                  18,263
rhel-server-rhscl-7-rpms/7Server/x86_64                                                                       Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server                                                    9,429
repolist: 28,003
```

**Review by lzap — APPROVED** [review]

File: plugins/katello/nightly/installation/index.md [hunk 1]
```diff
@@ -93,9 +93,10 @@ Installation may be done manually or via our recommended approach of using [fork
 {% highlight bash %}
 yum install -y yum-utils
 yum-config-manager --disable "*"
```
Comment (evgeni) [inline plugins/katello/nightly/installation/index.md:95]: do we need `yum-config-manager` at all then? seems we could remove that too

Comment (jhutar) [reply plugins/katello/nightly/installation/index.md:95]: @evgeni There is that `yum-config-manager --disable "*"` which is IMO useful/good practice

**Comment by mmoll** [comment]
@evgeni any objections left or is this good to go?

**Comment by evgeni** [comment]
It's definitely an improvement, so good to go

**Comment by mmoll** [comment]
merged, díky @jhutar!

**Pull request merged**